azavea/raster-vision
# Add Support for Shifting Rasters

## Overview

Modifies the window used in the `get_chip` method to "shift" input raster(s) by some number of meters in the x and/or y direction(s).  The rasters are not changed on disk.  Pursuant to #560 .

`[xy]_shift_meters` parameters have been [added to `GeoTiffFiles`](https://github.com/azavea/raster-vision/pull/573/files#diff-a38bff372bee7eadf2921734420052a1).

A positive x shift moves the "camera" to to the right.  A positive y shift moves the "camera" down.

### Checklist


### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as rebuilding the Docker image.
* Include test case, and expected output if not captured by automated tests.

Closes #https://github.com/azavea/raster-vision/issues/560
